Tchap/Modules/Favourites/FavouriteMessages/FavouriteMessagesCoordinatorType.swift

```diff
@@ -0,0 +1,29 @@
+// File created from ScreenTemplate
+// $ createScreen.sh Favourites/FavouriteMessages FavouriteMessages
+/*
+ Copyright 2020 New Vector Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import Foundation
+
+protocol FavouriteMessagesCoordinatorDelegate: class {
+    func favouriteMessagesCoordinator(_ coordinator: FavouriteMessagesCoordinatorType, didCompleteWithUserDisplayName userDisplayName: String?)
```

Comment:
I don't think `userDisplayName ` is actually expected here. Is it a mistake?

Reply:
It was generated by the template. I missed to remove it.